samples/MvcSample.Web/HomeController.cs

```diff
@@ -117,6 +119,8 @@ public ActionResult SaveUser(User user)
         /// </summary>
         public ActionResult Post([FromBody]User user)
         {
+            ViewData["Alive"] = true;
+            ViewData["Dependent.Alive"] = true;
             return View("MyView", user);
```

Comment:
Lets keep the home controller simple, we don't have to add this scenario to the sample.